go.mod

```diff
@@ -192,3 +206,5 @@ require (
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 	rsc.io/tmplfunc v0.0.3 // indirect
 )
+
+replace github.com/fbsobreira/gotron-sdk => github.com/smartcontractkit/chainlink-tron/relayer/gotron-sdk v0.0.5-0.20250422175525-b7575d96bd4d
```

Comment:
Where does this version come from? It looks like `relayer/v0.0.10` is `d7d29f6f0fc8a25943b01245a2a5ebc49d9c16aa`: https://github.com/smartcontractkit/chainlink-tron/releases/tag/relayer%2Fv0.0.10

Reply:
I think we just dropped the local replace, but the version that we are pointing to here still has it. The way to reproduce that local replace is to reference the same hash. Core was probably stale, because this has to be maintained with every bump of chainlink-tron. However, it is a very stale module, and so this likely doesn't actually make a difference.